src/v/cluster/feature_manager.cc

```diff
@@ -268,16 +270,18 @@ ss::future<> feature_manager::maybe_log_license_check_info() {
         }
     }
     if (_feature_table.local().is_active(features::feature::license)) {
-        if (license_required_feature_enabled()) {
+        auto enterprise_features = report_enterprise_features();
+        if (enterprise_features.any()) {
             const auto& license = _feature_table.local().get_license();
             if (!license || license->is_expired()) {
                 vlog(
                   clusterlog.warn,
-                  "Looks like you’ve enabled a Redpanda Enterprise feature(s) "
-                  "without a valid license. Please enter an active Redpanda "
-                  "license key (e.g. rpk cluster license set <key>). If you "
-                  "don’t have one, please request a new/trial license at "
-                  "https://redpanda.com/license-request");
+                  "Looks like you’ve enabled Redpanda Enterprise features "
+                  "([{}]) without a valid license. Please enter an active "
+                  "Redpanda license key (e.g. rpk cluster license set <key>). "
+                  "If you don’t have one, please request a new/trial license "
+                  "at https://redpanda.com/license-request",
+                  enterprise_features.enabled());
```

Comment:
Suggested text from docs team: 
A Redpanda Enterprise Edition license is required to use enterprise features: ([{}]). Enter an active license key (for example, rpk cluster license set <key>). To request a license, see https://redpanda.com/license-request. For more information, see https://docs.redpanda.com/current/get-started/licenses.

Reply:
Thank you. I've changed the messages to this now.

Reply:
On second thought, I think we should add the recovery instructions to the log message we print when the broker doesn't start up. I think your suggestion works for the license nag message printed periodically, but when we abort the broker, I think we should change the message above slightly and print the following instead:

A Redpanda Enterprise Edition license is required to use enterprise features: ([{}]). **Please downgrade this broker to the pre-upgrade version and** enter an active license key (for example, `rpk cluster license set <key>`). To request a license, see https://redpanda.com/license-request. For more information, see https://docs.redpanda.com/current/get-started/licenses.

What do you think @micheleRP?

Reply:
Thanks I've updated it to your suggestion now.